charter/index.html

```diff
@@ -28,269 +28,278 @@
+      where Charter is being developed.
+    </p>
     <ul>
-      <li>This Charter: <a href="https://w3c.github.io/webvr/charter/">https://w3c.github.io/webvr/charter/</a></li>
+      <li>This Charter: <a href=
+      "https://w3c.github.io/webvr/charter/">https://w3c.github.io/webvr/charter/</a>
+      </li>
       <li>Previous Charter: N/A
-      <li>Start Date: <span class="remove">{TBD: date the charter takes effect, estimate if not
-      known. Update this if the charter is revised and include a link to the
-      previous version of the charter.}</span></li>
-      <li>Last Modifed: <span class="remove">{TBD: If the system does not 
-      automatically provide information about
-      the date of the last modification, it can be useful to include that in
-      the charter.}</span></li>
+      </li>
+      <li>Start Date: <span class="remove">{TBD: date the charter takes effect,
+      estimate if not known. Update this if the charter is revised and include
+      a link to the previous version of the charter.}</span>
+      </li>
+      <li>Last Modifed: <span class="remove">{TBD: If the system does not
+      automatically provide information about the date of the last
+      modification, it can be useful to include that in the charter.}</span>
+      </li>
     </ul>
-
     <h2 id="goals">
       Goals
     </h2>
-
-    <p><span class="remove">{TBD: describe the mission and goals of the Community Group.
-    This should be a brief description describing the reason the group has been formed.}
-    </span></p>
-
+    <p>
+      The Goal of the W3C WebVR Community Group is to help bring
+      high-performance Virtual Reality to the open Web. Currently, access to VR
+      devices is limited to specific native platforms. For example, Oculus Rift
+      supports Windows and HTC Vive supports SteamVR platform only. With WebVR
+      specs, VR experiences could be available on the Web across platforms.
+    </p>
     <h2 id="scope-of-work">
       Scope of Work
     </h2>
-
-    <p class="remove">
-      {TBD: Describe topics that are in scope. For specifications that the CLA 
-      patent section applies to, it is helpful to describe the scope in a way 
-      that it is clear what types of technologies will be defined in specifications, 
-      as opposed to adoption by reference or underlying technology not defined in the 
-      proposed spec. Key use cases are often helpful in describing scope. If no 
-      specifications will be defined in the group that the CLA patent section applies 
-      to, the charter should clearly state that. A clear scope is particularly important 
-      where patent licensing obligations may apply.} 
-    </p>
-  
-      <h3 id="out-of-scope">
-        Out of Scope
-      </h3>
-    
-      <p class="remove">{TBD: Identify topics known in advance to be out of scope}</p>
-
+    <p>
+      One initial specification the group will work on is the WebVR API that
```

Comment:
@cvan ,  eventually should be on [w3c/webvr](https://github.com/w3c/webvr), but the spec hasn't been copied there yet.